ql/src/java/org/apache/hadoop/hive/ql/ddl/table/info/show/status/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Show table status DDL operation. */
```

Comment:
I don't see any use in these package-info files...they don't add any usefull content...
I think they could be removed...

Reply:
I agree, but the our checkstyle.xml tells us to put them there. I'll create a jira, and if we can agree to remove it from checkstyle.xml, then I'll remove all the package-info.java files from ddl.